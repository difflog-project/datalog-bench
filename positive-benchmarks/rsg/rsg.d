V: a,b,c,d,e,f,g,h,i,j,k,l,m,n,o,p.

*Up(V,V)
a,e
a,f
f,m
g,n
h,n
i,o
j,o
.
*Down(V,V)
l,f
m,f
g,b
h,c
i,d
p,k
.
*Flat(V,V)
g,f
m,n
m,o
p,m
.
Rsg(V,V)
g,f
m,n
m,o
p,m
a,b
f,k
h,f
i,f
j,f
a,c
a,d
.