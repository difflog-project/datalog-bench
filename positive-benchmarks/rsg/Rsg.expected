g	f
m	n
m	o
p	m
a	b
f	k
h	f
i	f
j	f
a	c
a	d
